content/en/catalyst/fund8/challenges/cross-chain-collaboration/campaign-brief.md

```diff
@@ -0,0 +1,89 @@
+How might we create connections and collaboration between Cardano and other blockchains in the next six months?
+
+### Why is it important?
+Interoperability of blockchains and multi-chain awareness guarantees shared success of the cryptoeconomy as a whole.
+
+### What does Sucess look like?
+Collaborative value creation between Project Catalyst and other communities. Both technical and non-technical initiatives are welcome.
+
+### Key Metrics to measure
+Objective metrics to measure success:
+
+- Mainnet and testnet transaction volume growth
+- Number of proposals including participants from other blockchain communities
+- Number of technical implementations connecting Cardano to other blockchains
+- Ideas from other chains implemented by Catalyst
+- Projects and dApps from other chains implemented on Cardano
+- Catalyst-created solutions adopted by other chains/communities
+- Number of events / workshops / sessions arranged with other communities
+- Number of permanent initiatives with cross-chain teams (members from different blockchain communities together)
+
+
+## Challenge Brief
+The success of Bitcoin spawned numerous off-shoot experiments and the arrival of Ethereum increased tribalism. As the number of altcoins grew, unhealthy "maximalism" emerged. The future is multi-chain, however, and the cryptoeconomy thrives when interoperability and cross-chain collaboration are cherished and encouraged.
+
+There are multiple next generation blockchains with cadres of zealous supporters, highly skilled developers and innovative technologies that Cardano and Project Catalyst can learn from. And, vice versa, we can help other networks thrive by offering the good practices developed in Catalyst. IOG has recently formed the UTXO alliance and started collaboration with numerous partners like COTI and SingularityNET. It's a good start but more is needed to address the lack of communication, coordination and collaboration between blockchain communities. Missing awareness of each other's doings forces us to duplicate our efforts. When we are separate we are not able to learn from each other's achievements or failures.
+
+Cardano has a strong, open and inclusive community, let's make this a real super power and let's be the first ones to forge strong bonds. Let's create an enriching, open and safe environment, where people from different communities can find common ground and share knowledge, resources, skills, ideas and visions with one another. Let's go out there, break the silos and build bridges and presence between us.
+
+**Here are some examples of potential DLT/Blockchain entities, communities and projects to explore and collaborate with:**
+
+**Cosmos**, dubbing itself the Internet of Blockchains, makes it easy for other blockchains to connect to it via their Inter-Blockchain Communication protocol (IBC). Massive projects like Binance Smart Chain, Terra, Thorchain and Oasis Network are built on the Cosmos SDK.
+
+- https://tendermint.com/sdk
```

Comment:
@PardusEidolon please turn these into markdown links for example instead of ```https://tendermint.com/sdk``` change it to [Tendermin SDK](https://tendermint.com/sdk)